src/Tags/Collection/Entries.php

```diff
@@ -309,7 +309,7 @@ protected function queryTaxonomies($query)
                 $values = array_filter(explode('|', $values));
             }
 
-            if (count($values) === 0) {
+            if ((is_countable($values) && count($values) === 0) || is_null($values)) {
```

Comment:
I don't think you need the is_countable. Just put the is_null first.

Reply:
Alright, swapped it around. I did think that was a bit clunky.